### PR TITLE
V0.2 cli admin tool including cluster status, job status and a terminal

### PR DESCRIPTION
**Pull request by zhawan**
# Description

Add the MARO admin tool.

MARO admin is a web UI for job management, including:

- A display window of the status of the job clusters
- A display window of the status of all jobs (running, pending, finished, failed, killed)
- An embedded terminal

Users can use the command "maro admin start/stop" to start or stop the MARO admin tool. And the web UI will be automatically launched to localhost:8080.

## Linked issue(s)/Pull request(s)

- [issue_number](issue_link)

## Type of Change

- [ ] Non-breaking bug fix
- [ ] Breaking bug fix
- [x] New feature
- [ ] Test
- [ ] Doc update
- [ ] Docker update

## Related Component

- [ ] Simulation toolkit
- [ ] RL toolkit
- [x] Distributed toolkit

## Has Been Tested

- OS:
  - [ ] Windows
  - [ ] Mac OS
  - [x] Linux
- Python version:
  - [ ] 3.6
  - [x] 3.7
- Key information snapshot(s):

## Needs Follow Up Actions

- [x] New release package
- [ ] New docker image

## Checklist

- [ ] Add/update the related comments
- [ ] Add/update the related tests
- [ ] Add/update the related documentations
- [ ] Update the dependent downstream modules usage

### CODE REVIEW

File: maro/cli/utils/web_terminal/monitor.py [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+                if attr_ref['chart_name'] in ["cpu", "gpu"]:
+                    chart_data[0] = chart_data[0] / 100
+
+                attr_ref['chart'].add_rows(chart_data)
+                attr_ref['len'] += data_len
+
+
+def write_job_list(root, list_name, list_data):
+    root.subheader(f"{list_name}:")
+    jqe = root.empty()
```
Comment (kaiqli) [inline maro/cli/utils/web_terminal/monitor.py:133]: better to use full name -> job_queue

File: maro/cli/utils/web_terminal/monitor.py [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+            list_str.append(f"<li>{data_str}</li>")
+        list_str.append("</ul>")
+        root.markdown("".join(list_str), True)
+
+
+def write_job_queue(root, data):
+    root.header("Job Queues:")
+    pjqe = write_job_list(root, "Pending Jobs", data["pending_jobs"])
+    kjqe = write_job_list(root, "Killed Jobs", data["killed_jobs"])
+    return pjqe, kjqe
```
Comment (kaiqli) [inline maro/cli/utils/web_terminal/monitor.py:154]: use full name

File: maro/cli/utils/web_terminal/monitor.py [hunk 3]
```diff
@@ -0,0 +1,366 @@
...
+def draw_dashboard_new(target):
+    if not target:
+        return
+
+    try:
+        static_node_load = False
+        dynamic_node_load = False
+
+        while True:
+
```
Comment (kaiqli) [inline maro/cli/utils/web_terminal/monitor.py:305]: rm blank line

File: maro/cli/utils/web_terminal/monitor.py [hunk 4]
```diff
@@ -0,0 +1,366 @@
...
+                job_queue_data = local_executor.get_job_queue()
+                pjqe, kjqe = write_job_queue(st, job_queue_data)
+
+                st.markdown("---")
+
+                job_detail_data = local_executor.get_job_details()
+                jde = write_job_details(st, job_detail_data, dashboard_type)
+
+                dynamic_node_load = True
+
```
Comment (kaiqli) [inline maro/cli/utils/web_terminal/monitor.py:335]: rm blank line

File: maro/cli/utils/web_terminal/monitor.py [hunk 5]
```diff
@@ -0,0 +1,366 @@
...
+def main():
+    target = None
+
+    with st.spinner("Loading Clusters..."):
+        clusters = load_clusters()
+        for cluster_name in clusters:
+            if st.sidebar.button(f'{cluster_name}'):
+                target = cluster_name
+    # with st.empty():
```
Comment (kaiqli) [inline maro/cli/utils/web_terminal/monitor.py:361]: rm if no need

File: maro/cli/utils/web_terminal/requirements.txt [hunk 6]
```diff
@@ -0,0 +1,16 @@
+#
```
Comment (kaiqli) [inline maro/cli/utils/web_terminal/requirements.txt:1]: Does need merge to maro/requirements.dev.txt?

File: maro/cli/utils/web_terminal/monitor.py (hunk 1 above)
Comment (zhawan) [reply maro/cli/utils/web_terminal/monitor.py]: updated

File: maro/cli/utils/web_terminal/monitor.py (hunk 2 above)
Comment (zhawan) [reply maro/cli/utils/web_terminal/monitor.py]: updated

File: maro/cli/utils/web_terminal/monitor.py (hunk 3 above)
Comment (zhawan) [reply maro/cli/utils/web_terminal/monitor.py]: removed

File: maro/cli/utils/web_terminal/monitor.py (hunk 4 above)
Comment (zhawan) [reply maro/cli/utils/web_terminal/monitor.py:333]: removed

File: maro/cli/utils/web_terminal/monitor.py (hunk 5 above)
Comment (zhawan) [reply maro/cli/utils/web_terminal/monitor.py]: removed

File: maro/cli/utils/web_terminal/requirements.txt (hunk 6 above)
Comment (zhawan) [reply maro/cli/utils/web_terminal/requirements.txt:1]: MARO did not import admin tool, seems admin tool's requirements are not need to be included in MARO.
add a  "maro admin req" command to install requirements for CLI admin tool

**Review by Jinyu-W — COMMENTED** [review]
refine the PR title and description. Add more details about the changes (including what bug fix if it contains)

**Review by Jinyu-W — APPROVED** [review]

**Pull request merged**